react_projects/rick-and-morty-web/src/components/rawItem/RawItem.js

```diff
@@ -0,0 +1,26 @@
+import './RawItem.css'
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+
+class RawItem extends Component {
+  render() {
+    let { character } = this.props
+    return (
+      <Link to={`/characters/${character.id}`} className='link-disabled'>
```

Comment:
Having a routes file prevents you from having possible values when you repeat the route explicitly